src/wodin.ts

```diff
@@ -0,0 +1,30 @@
+import * as base from "./base";
+import type { OdinModelConstructable, Solution } from "./model";
+import {runModel} from "./model";
+import type { UserType } from "./user";
+
+// tslint:disable-next-line:variable-name
+export function wodinRun(Model: OdinModelConstructable, pars: UserType,
```

Comment:
Not sure why this would trigger that linting rule?

Reply:
It objects to `Model` as a variable name

Reply:
Ah right, sorry I was just looking at wodinRunner itself. Right, and it makes sense to PascalCase that as it's a class.